autogpt_platform/backend/backend/data/graph.py

```diff
@@ -190,14 +197,19 @@ def output_schema(self) -> dict[str, Any]:
             )
         )
 
+    @computed_field
+    @property
+    def credentials_input_schema(self) -> dict[str, Any]:
```

Comment:
Seeing the usage of this, why don't we just return the BlockSchema and use (or add more) fields to access it instead of a raw dictionary?

Reply:
What do you mean? I use `Graph.credentials_input_schema` in the frontend to render the credentials inputs.

Reply:
I did not know this is a property field for the autogenerating field for the api.